pkg/util/containers/providers/windows/provider.go

```diff
@@ -144,8 +144,23 @@ func (mp *provider) GetContainerMetrics(containerID string) (*metrics.ContainerM
 
 // GetContainerLimits returns CPU, Thread and Memory limits
 func (mp *provider) GetContainerLimits(containerID string) (*metrics.ContainerLimits, error) {
-	// FIXME: Figure out a way to extract limits from Job Objects in the Windows Kernel
-	return nil, fmt.Errorf("not supported on windows")
+	dockerUtil, err := docker.GetDockerUtil()
+	if err != nil {
+		return nil, err
+	}
+
+	cjson, err := dockerUtil.Inspect(containerID, false)
+	if err != nil {
+		return nil, err
+	}
+
+	containerLimits := metrics.ContainerLimits{
+		CPULimit:    float64(cjson.HostConfig.CPUQuota),
```

Comment:
I believe this information is in `HostConfig.CpuShares` or `HostConfig.NanoCpus`.
`CpuShares` is a relative number between 1 and 10000 that represents the share of CPU this container should receive (compared to other containers).
`NanoCpus` is absolute number, expressed in nano CPU Cores, e.g. "NanoCpus": 1500000000 = 1.5 CPU Cores